elfi/methods/bo/acquisition.py

```diff
@@ -587,8 +587,9 @@ def __init__(self, quantile_eps=.01, integration='grid', d_grid=.2,
         quantile_eps : int, optional
             Quantile of the observed discrepancies used in setting the discrepancy threshold.
         integration : str, optional
-            Integration method.
-            (Options: importance, grid.)
+            Integration method. Options:
+            - grid: more accurate yet computationally expensive in high dimensions;
+            - importance: less accurate though applicable in high dimensions.
         d_grid : float, optional
             Grid tightness.
```

Comment:
Maybe explain this a bit more, what does this options actually do. Wouldn't it make more sense to allow setting the amount of grid points instead of grid element length which depends on the scaling of parameter space?

Reply:
in my opinion, it makes more sense to have the length since the high number of points will not necessarily mean that the grid is tight enough for a close approximation.

Reply:
Ok, let's let it be this way then. However, if the scales of the parameters were different, then it would certainly make more sense to set the amount of grid points or specify different lengths (though, this is not relevant here because the GP model used), or if random points from uniform density are used.